test/test-cases/react-css-modules/app/src/App.js

```diff
@@ -2,13 +2,21 @@ import React from 'react';
 import lessStyles from './lessStyles.less';
 import jsStyles from './jsStyles.css.js';
 
-export default () => (
-  <div className={`${lessStyles.root} ${jsStyles.root}`}>
-    <div
-      className={`${lessStyles.nested} ${jsStyles.nested}`}
-      data-automation-text
-    >
-      Hello World
+export default () => {
+  const [message, setMessage] = React.useState('Initial render');
+
+  setTimeout(() => {
+    setMessage('Updated render');
```

Comment:
Would be worth adding a comment here to explain why—otherwise it looks a bit odd.